ouroboros-consensus/test/consensus-test/Test/Consensus/MiniProtocol/LocalStateQuery/Server.hs

```diff
@@ -216,7 +217,7 @@ initLgrDB k chain = do
     blockMapping = Map.fromList
       [(blockRealPoint b, b) | b <- Chain.toOldestFirst chain]
 
-    cfg = configLedgerDb $ testCfg k
+    cfg = configLedgerDb (testCfg k) (ApplyOpts ValidationOn OmitLedgerEvents)
```

Comment:
I wonder if it'd be worth factoring out a `defaultApplyOpts` and using that everywhere this triplet occurs?

Reply:
No longer applicable